com.milesalan.mepo.yml

```diff
@@ -0,0 +1,110 @@
+# Flatpak Manifest
+app-id: com.milesalan.mepo
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: ["org.freedesktop.Sdk.Extension.ziglang"]
+extension-tag: '21.08'
```

Comment:
```suggestion
```

you don't need this

Reply:
Fixed

Reply:
Hmm seems like build was failing without this

Reply:
this is weird.